pkgs/applications/graphics/xournal/default.nix

```diff
@@ -25,5 +25,6 @@ stdenv.mkDerivation rec {
     homepage = http://xournal.sourceforge.net/;
     description = "note-taking application (supposes stylus)";
     maintainers = [ stdenv.lib.maintainers.guibert ];
+    licenses = [ stdenv.lib.licenses.gpl2 ];
```

Comment:
This should be `license`. There are a few other pkgs which use `licenses`, but I cannot find anything suggesting it's a valid alias.